File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int reverse = 0;
        int temp = x;
        while(temp>0){
            int rem = temp % 10;
            reverse = (reverse * 10) + rem;
            temp /= 10;
        }
        if(x == reverse){
            return true;
        }      
        else{
            return false;
        }
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        original = x
        rev = 0
        while(x > 0):
            last = x%10
            rev = rev * 10 + last
            x = x//10
        
        return rev == original
